apps/openchallenges/notebook/src/challenge_headline/generate_challenge_headlines.py

```diff
@@ -80,8 +80,11 @@
 
 def generate_challenge_headlines(text, num_headlines):
     prompt = (
-        f"Please generate {num_headlines} headlines that have a maximum ten words from the "
-        "following challenge description. The headline must summarize the goal of the challenge. "
+        f"Please generate {num_headlines} headlines that have less than 80 characters from the "
+        "following challenge description. "
+        "The headlines must summarize the goal of the challenge. "
+        # "The headlines must not include the name of the challenge. "
+        "The headlines must reads naturally. "
```

Comment:
```suggestion
        "The headlines must read naturally. "
```

^ this probably won't have an effect on the end result, but in case it does...